src/gui/georeferencing_dialog.cpp

```diff
@@ -48,13 +48,11 @@
 #include <QPushButton>
 #include <QRadioButton>
 #include <QSignalBlocker>
-#include <QSize>
 #include <QSpacerItem>
 #include <QStringRef>
 #include <QTimer>
 #include <QUrl>
 #include <QUrlQuery>
-#include <QVariant>
```

Comment:
I guess this is still used, but I can't check now.

Reply:
You are also right, it's used by Settings::getInstance().getSetting().